share/p2p/peers/manager.go

```diff
@@ -152,31 +159,40 @@ func (s *Manager) Peer(
 	ctx context.Context, datahash share.DataHash,
 ) (peer.ID, DoneFunc, error) {
 	p := s.getOrCreatePool(datahash.String())
-	p.markValidated()
+	if p.markValidated() {
+		log.Debugw("marked validated", "datahash", datahash.String())
+	}
 
 	// first, check if a peer is available for the given datahash
 	peerID, ok := p.tryGet()
 	if ok {
 		// some pools could still have blacklisted peers in storage
 		if s.peerIsBlacklisted(peerID) {
+			log.Debugw("blacklisting bad shrex-sub peer", "hash", datahash.String(),
+				"peer", peerID.String())
```

Comment:
Would it make more sense to start with the peer ID?
```suggestion
			log.Debugw("blacklisting bad shrex-sub peer", "peer", peerID.String(), "hash", datahash.String)
```

Reply:
I kept same order of log fields everywhere for easy `grep`